springwolf-core/src/main/java/io/github/stavshamir/springwolf/schemas/example/ExampleXmlValueGenerator.java

```diff
@@ -217,7 +192,7 @@ public String serializeIfNeeded(String name, Node exampleObject) {
         try {
             document.appendChild(objectToWrite);
             String xml = exampleXmlValueSerializer.writeDocumentAsXmlString(document);
-            log.info("name {} -> xml: {}", name, xml);
+            log.debug("name {} -> xml: {}", name, xml);
```

Comment:
If we want to keep the debug log, maybe we should add more context to the log statement